### ISSUE TITLE
End-user documentation

### ISSUE THREAD

**Issue by willb** [ophicleide/ophicleide-training#12]
Create a GitHub Pages repo for end-user documentation for Ophicleide.

**Comment by elmiko** [comment]
i'm thinking this will be something top-level for the organization, like we could make ophicleide/ophicleide-pages or similar.

**Comment by willb** [comment]
yeah, agreed, just capturing it here while I was thinking about it
